docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,338 @@
+
+There is also a distinction between `New Users` who have not previously had a session on the website (domain_sessionidx=1) and `Existing users` who have a session with domain_sessionidx > 1.
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_AO.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_AO.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_AO.png)
+
+</TabItem>
+
+</Tabs>
+
+### User Acquisition
+This dashboard is about **User Acquisition** which is effectively about what channels / methods were used to acquire new users.
+
+Every visual is already filtered on new users only (domain_sessionidx = 1).
+
+Metrics definitions:
+
+**engaged_sessions** = count(case when engaged=True end)
+**engagement_rate** = engaged_sessions / count(distinct domain_sessionid)
+**conversion_count** =sum(cv__all_volume)
+**conversion_revenue** = sum(cv__all_total)
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_UA.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_UA.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_UA.png)
+
+</TabItem>
+
+</Tabs>
+
+### Traffic Acquisition
+
+This dashboard is about **Traffic Acquisition** which aims at highlighting which channels are used to re-engage existing (returning) users.
+
+Every visual is already filtered on returning users only (domain_sessionidx =>1).
+
+Metrics definitions:
+
+**engaged_sessions** = count(case when engaged=True end)
+**engagement_rate** = engaged_sessions / count(distinct domain_sessionid)
+**conversion_count** =sum(cv__all_volume)
+**conversion_revenue** = sum(cv__all_total)
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_TA.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_TA.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_TA.png)
+
+</TabItem>
+
+</Tabs>
+
+### Entrance and Exit Points
+
+A separate overview is dedicated for the top 10 Entrance and Exit points with a breakdown on new and returning users.
+
+An `entrance point` is the first page URL within a session. Although the sessions table produces a field called first_page_url (first_page_url_scheme  || '://' || first_page_url_hist || first_page_url_path), for BI purposes a calculated field is being used on this dashboard to exclude GTM tags.
+
+Similarly an `exit point` is the last page URL within a session. It is calculated using last_page_url_scheme  || '://' || last_page_url_host || last_page_url_path.
+
```

Comment:
I don't think we need to explain this in this much detail, it's probably enough to say we remove any query or fragment parts from the url.